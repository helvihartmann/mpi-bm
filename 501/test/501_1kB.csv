"send","receive"
"0.024938","0.024972"
"0.027190","0.027185"
"0.000025","0.000040"
"0.000001","0.000001"
"0.000011","0.000009"
"0.000002","0.000003"
"0.000002","0.000001"
"0.000001","0.000002"
"0.000002","0.000002"
"0.000002","0.000002"
"0.000203","0.000205"
"0.000001","0.000001"
"0.000001","0.000001"
"0.000001","0.000001"
"0.000001","0.000001"
"0.000001","0.000001"
"0.000001","0.000001"
"0.000001","0.000070"
"0.000001","0.000005"
"0.000001","0.000000"
"0.000001","0.000000"
"0.000001","0.000000"
"0.000001","0.000000"
"0.000000","0.000000"
"0.000001","0.000001"
"0.000001","0.000001"
"0.000000","0.000001"
"0.000001","0.000008"
"0.000001","0.000000"
"0.000001","0.000000"
"0.000001","0.000000"
"0.000000","0.000000"
"0.000001","0.000001"
"0.000001","0.000001"
"0.000000","0.000001"
"0.000001","0.000000"
"0.000001","0.000000"
"0.000001","0.000004"
"0.000001","0.000001"
"0.000001","0.000000"
"0.000001","0.000000"
"0.000000","0.000000"
"0.000217","0.000147"
"0.000003","0.000007"
"0.000002","0.000005"
"0.000001","0.000004"
"0.000001","0.000004"
"0.000001","0.000014"
"0.000001","0.000001"
"0.000000","0.000001"
"0.000001","0.000001"
"0.000001","0.000001"
"0.000001","0.000001"
"0.000001","0.000002"
"0.000000","0.000001"
"0.000001","0.000001"
"0.000001","0.000001"
"0.000001","0.000001"
"0.000000","0.000001"
"0.000001","0.000002"
"0.000001","0.000001"
"0.000000","0.000001"
"0.000001","0.000001"
"0.000000","0.000001"
"0.000001","0.000001"
"0.000001","0.000001"
"0.000016","0.000010"
"0.000001","0.000005"
"0.000000","0.000001"
"0.000001","0.000001"
"0.000000","0.000001"
"0.000001","0.000001"
"0.000000","0.000001"
"0.000001","0.000001"
"0.000000","0.000001"
"0.000001","0.000001"
"0.000000","0.000001"
"0.000001","0.000001"
"0.000000","0.000001"
"0.000001","0.000001"
"0.000000","0.000001"
"0.000001","0.000002"
"0.000001","0.000000"
"0.000001","0.000002"
"0.000000","0.000000"
"0.000001","0.000002"
"0.000001","0.000000"
"0.000000","0.000001"
"0.000001","0.000000"
"0.000000","0.000002"
"0.000001","0.000001"
"0.000000","0.000002"
"0.000001","0.000001"
"0.000001","0.000002"
"0.000001","0.000000"
"0.000000","0.000002"
"0.000001","0.000000"
"0.000001","0.000002"
"0.000000","0.000000"
