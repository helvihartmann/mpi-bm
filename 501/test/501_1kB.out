1000 Byte are send
0.024938 seconds for Process 0 to send.
0.027190 seconds for Process 0 to send.
0.000025 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000011 seconds for Process 0 to send.
0.000002 seconds for Process 0 to send.
0.000002 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000002 seconds for Process 0 to send.
0.000002 seconds for Process 0 to send.
0.000203 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000217 seconds for Process 0 to send.
0.000003 seconds for Process 0 to send.
0.000002 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000016 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000001 seconds for Process 0 to send.
0.000000 seconds for Process 0 to send.
0.024972 seconds for Process 1 to receive.
0.027185 seconds for Process 1 to receive.
0.000040 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000009 seconds for Process 1 to receive.
0.000003 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000002 seconds for Process 1 to receive.
0.000002 seconds for Process 1 to receive.
0.000002 seconds for Process 1 to receive.
0.000205 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000070 seconds for Process 1 to receive.
0.000005 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000008 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000004 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000147 seconds for Process 1 to receive.
0.000007 seconds for Process 1 to receive.
0.000005 seconds for Process 1 to receive.
0.000004 seconds for Process 1 to receive.
0.000004 seconds for Process 1 to receive.
0.000014 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000002 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000002 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000010 seconds for Process 1 to receive.
0.000005 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000002 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000002 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000002 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000002 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000002 seconds for Process 1 to receive.
0.000001 seconds for Process 1 to receive.
0.000002 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000002 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.
0.000002 seconds for Process 1 to receive.
0.000000 seconds for Process 1 to receive.

everything send correctly 
